Source/Core/DolphinQt/Achievements/AchievementProgressWidget.cpp

```diff
@@ -42,124 +38,53 @@ AchievementProgressWidget::AchievementProgressWidget(QWidget* parent) : QWidget(
   setLayout(layout);
 }
 
-QGroupBox*
-AchievementProgressWidget::CreateAchievementBox(const rc_api_achievement_definition_t* achievement)
+void AchievementProgressWidget::UpdateData(bool clean_all)
 {
-  const auto& instance = AchievementManager::GetInstance();
-  if (!instance.IsGameLoaded())
-    return new QGroupBox();
-
-  QLabel* a_badge = new QLabel();
-  const auto unlock_status = instance.GetUnlockStatus(achievement->id);
-  const AchievementManager::BadgeStatus* badge = &unlock_status.locked_badge;
-  std::string_view color = AchievementManager::GRAY;
-  if (unlock_status.remote_unlock_status == AchievementManager::UnlockStatus::UnlockType::HARDCORE)
-  {
-    badge = &unlock_status.unlocked_badge;
-    color = AchievementManager::GOLD;
-  }
-  else if (hardcore_mode_enabled && unlock_status.session_unlock_count > 1)
```

Comment:
This removal leaves the constexpr static variable `hardcore_mode_enabled` unused, and I think the point of it was for testing anyway, so maybe remove it.